### DISCUSSION TITLE
5.0 Released

### DISCUSSION THREAD

**Discussion by zmstone** [emqx/emqx#8272] [Show and tell]
After branched off from 4.3.3 on June.5 2021, 5.0 collected more than 5,000 commits.
Thanks a lot EMQX team members and contributors.
And all the open source library authors.

release note here:
https://github.com/emqx/emqx/releases/tag/v5.0.0

doc: https://www.emqx.io/docs/en/v5.0

**Comment by electrocoder** [comment]
With 5.0, the plugin list was removed. It is necessary to install the .tar.gz package, but there is no 5.0 in the documentation.

**Reply by zmstone** [reply]
Hi @electrocoder 
Thank you for your interest.
The 5.0 document is not quite ready yet.
For plugin management, there is dashboard, also CLI.
For development, you can refer to https://github.com/emqx/emqx-plugin-template

**Reply by PhiRie** [reply]
@zmstone 
would also be interested in the new v5.0 documentation. Especially the changes for the helm deployments e.g. emqxAclConfig, emqxLoadedModules and emqxLoadedPlugins. Looking forward!

**Comment by zmstone** [comment]
hi @PhiRie 
the documents are almost ready, some translation & review work left, you can find the content here.
https://github.com/emqx/emqx-docs/tree/release-5.0
the full configuration doc can be previewed directly on github because it's markdown
https://github.com/emqx/emqx-docs/blob/release-5.0/en_US/admin/cfg.md

a quick answer to your listed questios
* acl.conf is kept backward compatible, and it's by default enabled. now you can edit from the dashboard UI.
* loaded_module and loaded_plugins configs are removed, instead, there is a `enable` field in each feature.
  for example `delayed.enable = false` to disable the delayed publish feature.

**Comment by MarcoPai** [comment]
您好，CentOS-7 通过yum安装5.0之后，目录结构和文档里面的对不上。

**Reply by MarcoPai** [reply]
可否提供5.0的目录结构，万分感谢

**Reply by zmstone** [reply]
The doc isn’t released yet, but here is the raw content.
https://github.com/emqx/emqx-docs/blob/release-5.0/en_US/deploy/dirs.md

**Comment by amnonbb** [comment]
Can be v4.x.x and 5.x.x in same cluster?

**Reply by zmstone** [reply]
Hi @amnonbb 
Unfortunately not a rolling upgrade this time.
Our plan going forward: 6.0 will be able to form cluster with 5.x.

**Comment by zmstone** [comment]
doc released: https://www.emqx.io/docs/en/v5.0
